subd0my.py

```diff
@@ -1,44 +1,83 @@
 #!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
-print("\033[1;30;40m           _         _  ___                  ")
-print("\033[1;30;40m ___ _   _| |__   __| |/ _ \ _ __ ___  _   _ ")
-print("\033[1;30;40m/ __| | | | '_ \ / _` | | | | '_ ` _ \| | | |")
-print("\033[1;30;40m\__ \ |_| | |_) | (_| | |_| | | | | | | |_| |")
-print("\033[1;30;40m|___/\__,_|_.__/ \__,_|\___/|_| |_| |_|\__, |")
-print("\033[1;30;40m                                        |___/")
-print("")
-print("")
+# standard library modules
+import sys
+import argparse
 
+# external modules
 import requests
+from colorama import Fore
+
+banner = """
+               _         _  ___                  
+     ___ _   _| |__   __| |/ _ \ _ __ ___  _   _ 
+    / __| | | | '_ \ / _` | | | | '_ ` _ \| | | |
+    \__ \ |_| | |_) | (_| | |_| | | | | | | |_| |
+    |___/\__,_|_.__/ \__,_|\___/|_| |_| |_|\__, |
+                                            |___/
+"""
+print(Fore.GREEN + banner)
+print("")
```

Comment:
remove this why issue another command for a single character?
```suggestion
print("")
```